R/derive_vars_disposition_reason.R

```diff
@@ -261,15 +265,18 @@ format_reason_default <- function(reason, reason_spe = NULL) {
 #' `reason_var` value (e.g. `if_else ( reason != "COMPLETED" & !is.na(reason), reason_spe,
 #' NA_character_)`).
 #' `DCSREASP = format_reason(DSDECOD, DSTERM)` returns DCSREASP = `DSTERM` when `DSDECOD` is not
-#' 'COMPLETED' nor NA.
+#' 'COMPLETED' nor NA, and `DSDECOD` is equal to "OTHER".
```

Comment:
Please use backticks for `"OTHER"`.

Should the comma be removed?

Reply:
I also updated other parts similar to 'OTHER' that does not have backticks (eg 'COMPLETED'). Please review again to see if it is ok.